cypress/e2e/pages/viewApplication.cy.ts

```diff
@@ -69,4 +69,75 @@ describe('View a resident application', () => {
     ViewApplicationPage.getErrorSummary().should('be.visible');
     cy.contains(`Unable to delete household member`);
   });
+
+  it('shows the notes and history section for read only users', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application);
+
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection().should('exist', {
+      timeout: 5000,
+    });
+  });
+
+  it('allows readonly users to see note section', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId, true);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application, true);
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection()
+      .click()
+      .get('[data-testid="add-text-input"]')
```

Comment:
if you do need the text input id selector it should probably be in the page model. I'm pretty sure if you look in the development branch you'll see a components page model that will point to this element.